### PR TITLE
EES-1076 Remove redundant types & stored procs following datafactory …

### PR DESCRIPTION
**Pull request by shakes90**
…changes & add stored procedure to recreate the Release row.

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_Routine_DropAndCreateRelease.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CREATE PROCEDURE DropAndCreateRelease @Release dbo.ReleaseType READONLY
```
Comment (benoutram) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_Routine_DropAndCreateRelease.sql]: This file isn't run

File: src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_RemoveRedundantProcs.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterGroup");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterGroupFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterItem");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterItemFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertIndicator");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertIndicatorFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertIndicatorGroup");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertSubject");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertSubjectFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertRelease");
```
Comment (benoutram) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_RemoveRedundantProcs.cs]: This isn't the full list but I'm presuming that's intentional? The following remain:

- UpsertFootnote,
- UpsertTheme
- UpsertTopic
- UpsertPublication
- UpsertLocation

File: src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_Routine_DropAndCreateRelease.sql (hunk 1 above)
Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_Routine_DropAndCreateRelease.sql]: Hah well spotted sir - teach me to add things to the db manually!

File: src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_RemoveRedundantProcs.cs (hunk 2 above)
Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_RemoveRedundantProcs.cs]: Yes intentional - only needed for the tables where I need to upsert - the release is deleted on re-run so all related tables can just use standard data factory copy.

Comment (benoutram) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_RemoveRedundantProcs.cs]: Great thanks

File: src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_Routine_DropAndCreateRelease.sql (hunk 1 above)
Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_Routine_DropAndCreateRelease.sql]: OK fxied

**Pull request merged**